plugins/inara.py

```diff
@@ -757,30 +954,42 @@ def journal_entry(cmdr, is_beta, system, station, entry, state):
         this.system_link.update_idletasks()
 
     if config.get('station_provider') == 'Inara':
-        this.station_link['text'] = this.station or (this.system_population and this.system_population > 0 and STATION_UNDOCKED or '')
+        to_set = this.station
+        if not to_set and this.system_population is not None:
+            if this.system_population > 0:
+                to_set = STATION_UNDOCKED
+            else:
+                to_set = ''
```

Comment:
This doesn't look equivalent.  Only the `this.system_population > 0` should be gated by the `this.system_population is not None`.  The latter is in case we plain don't have that variable set yet.  It shouldn't be preventing the else doing `to_set = ''`

Reply:
Whoops. yeah. maybe:
```py
if not to_set:
    if this.system_population is not None and this.system_population > 0:
        to_set = STATION_UNDOCKED
    else:
        to_set = ''
```